src/Okta.Sdk.UnitTests/FeaturesClientShould.cs

```diff
@@ -0,0 +1,36 @@
+﻿// <copyright file="FeaturesClientShould.cs" company="Okta, Inc">
+// Copyright (c) 2020 - present Okta, Inc. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+// </copyright>
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using NSubstitute;
+using Okta.Sdk.Internal;
+using Okta.Sdk.UnitTests.Internal;
+using Xunit;
+
+namespace Okta.Sdk.UnitTests
+{
+    public class FeaturesClientShould
+    {
+        [Fact]
+        public async Task CallUpdateFeatureLifeCycleEndpoint()
```

Comment:
Due to concerns with tests that modify data (considering potential for server related issues), I opted to implement this test as a unit test using mocks instead of a more detailed integration test.  We can certainly revisit the creation of integration tests for this as time allows.

Reply:
I know we have been struggling with lots of IT issues. However, our default should still be creating ITs unless it is too complicated or we are actually seeing issues that are difficult to solve. The idea is that we help to identify problems with our API.

Reply:
I review further.

Reply:
Now that I know that there's no API provided to create our own feature and play with it, I think you made a good decision on writing a UT for this case instead 👍 .